File: Belot/Models/Http/Requests/SignalR/HandAnnouncementRequest.cs
using Belot.Models.Belot;

namespace Belot.Models.Http.Requests.SignalR
{
    public class HandAnnouncementRequest : BaseSignalRRequest
    {
        public HandAnnouncement Announcement { get; set; }
        public int? HighestRank { get; set; } 
    }
}

File: Belot/Belot.Client/src/app/server-api/requests/signalR/hand-announcement-request.ts
import { HandAnnouncementType } from "../../../BelotEngine/Announcement";
import BaseSignalRRequest from "./base-signalr-request";

export default class HandAnnouncementRequest extends BaseSignalRRequest {
  announcement!: HandAnnouncementType;
  highestRank!: number;
}
